nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -111,6 +111,8 @@ In addition to numerous new and upgraded packages, this release has the followin
 
   * The data directory remains located at `/var/lib/bitwarden_rs`, for backwards compatibility.
 
+* Python package `pytest-rerunfailures` was upgraded to a new major release. See [upstream changelog](https://github.com/pytest-dev/pytest-rerunfailures/blob/10.1/CHANGES.rst#101-2021-07-02).
+
```

Comment:
I've interpreted https://github.com/NixOS/nixpkgs/blob/c95467f1bf721f15a459286fda454f6f946bc5bc/.github/PULL_REQUEST_TEMPLATE.md#L26-L27 to apply to major version bumps of any package, i.e. including this one here. Dunno if that's a good idea though, as it would make the NixOS release notes quite long and verbose.

Reply:
I may need to clarify this for packages exposed in all-packages.nix.

The desire of the that clause is for people upgrading to be able to anticipate a breakage. I don't think many will care if a package within an ecosystem gets bumped.